packages/nimble-components/src/table/template.ts

```diff
@@ -48,23 +49,38 @@ export const template = html<Table>`
                      role="rowgroup">
                     ${when(x => x.columns.length > 0 && x.canRenderRows, html<Table>`
                         ${repeat(x => x.virtualizer.visibleItems, html<VirtualItem, Table>`
-                            <${tableRowTag}
-                                class="row"
-                                record-id="${(x, c) => c.parent.tableData[x.index]?.id}"
-                                :dataRecord="${(x, c) => c.parent.tableData[x.index]?.record}"
-                                :columns="${(_, c) => c.parent.columns}"
-                                @row-action-menu-beforetoggle="${(_, c) => c.parent.onRowActionMenuBeforeToggle(c.event as CustomEvent<TableActionMenuToggleEventDetail>)}"
-                                @row-action-menu-toggle="${(_, c) => c.parent.onRowActionMenuToggle(c.event as CustomEvent<TableActionMenuToggleEventDetail>)}"
-                            >
-                            ${when((x, c) => (c.parent as Table).openActionMenuRecordId === (c.parent as Table).tableData[x.index]?.id, html<VirtualItem, Table>`
-                                ${repeat((_, c) => (c.parent as Table).actionMenuSlots, html<string, Table>`
-                                    <slot
-                                        name="${x => x}"
-                                        slot="${x => `row-action-menu-${x}`}">
-                                    </slot>
-                                `)}
-                            `)}                        
-                            </${tableRowTag}>
+                            ${when((x, c) => (c.parent as Table).tableData[x.index]?.isGrouped, html<VirtualItem, Table>`
```

Comment:
Shouldn't the header row shift over when grouping is enabled? Are you planning on addressing that in a different PR (i.e. one that adds the collapse-all button)?

If you are explicitly not including that in this PR, it would be good to include that in the "This PR does not include" list in the PR description.

Reply:
Yeah. my intent was to address that when introducing the collapse-all button. I'll update the description.